test/aircraft/AircraftCollection.spec.js

```diff
@@ -0,0 +1,79 @@
+import ava from 'ava';
+import AircraftCollection from '../../src/assets/scripts/client/aircraft/AircraftCollection';
+import { ARRIVAL_AIRCRAFT_MODEL_MOCK, DEPARTURE_AIRCRAFT_MODEL_MOCK } from './_mocks/aircraftMocks';
+
+ava('does not throw with valid parameters', (t) => {
+    t.notThrows(() => new AircraftCollection());
+});
+
+ava('.add() throws when passed invalid params', (t) => {
+    const collection = new AircraftCollection();
+
+    t.throws(() => collection.add(null));
+});
+
+ava('.add() increases the `#_items` length by 1', (t) => {
+    const collection = new AircraftCollection();
+
+    collection.add(ARRIVAL_AIRCRAFT_MODEL_MOCK);
+
+    t.true(collection.length === 1);
+});
+
+ava('.remove() throws when the itemToRemove does not exist in `#_items`', (t) => {
+    const collection = new AircraftCollection();
+
+    collection.add(DEPARTURE_AIRCRAFT_MODEL_MOCK);
```

Comment:
does the collection accept non-instantiated `AircraftModel` objects?  reading this test it seems like this is just data you're setting to the collection.

for a test like this, I'm cool with directly assigning to items and then running the assertion with `remove`